<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-09QGL8M" name="com.steerlean.scenarios.pet.PetDELETE" tests="6" failures="1" timestamp="17 Dec 2020 06:38:20 GMT" time="20.120" errors="0">
  <testcase name="TC_05" time="2.993" classname="com.steerlean.scenarios.pet.PetDELETE"/>
  <testcase name="TC_03" time="3.074" classname="com.steerlean.scenarios.pet.PetDELETE"/>
  <testcase name="TC_01" time="4.933" classname="com.steerlean.scenarios.pet.PetDELETE">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.fail(Assert.java:103)
at com.steerlean.scenarios.pet.PetDELETE.success(PetDELETE.java:132)
at com.steerlean.scenarios.pet.PetDELETE.TC_01(PetDELETE.java:47)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TC_01 -->
  <testcase name="TC_04" time="3.254" classname="com.steerlean.scenarios.pet.PetDELETE"/>
  <testcase name="TC_06" time="3.083" classname="com.steerlean.scenarios.pet.PetDELETE"/>
  <testcase name="TC_02" time="2.783" classname="com.steerlean.scenarios.pet.PetDELETE"/>
</testsuite> <!-- com.steerlean.scenarios.pet.PetDELETE -->
